recipes/async_generator/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "async_generator" %}
+{% set version = "1.9" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "b7d5465c6174fe86dba498ececb175f93a6097ffb7cc91946405e1f05b848371" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
```

Comment:
We are moving to ``script: python -m pip install --no-deps --ignore-installed .``. See https://github.com/conda-forge/conda-forge.github.io/pull/518

Reply:
Done